### PR TITLE
Version bump + yamlgen

### PR DESCRIPTION
**Pull request by GregSutcliffe**

### CODE REVIEW

File: generate_answers.rb [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+# Bonus, per-module, questions defined up top where they're easy to find :)
+
+def foreman_questions
+  {
+    "Should Foreman require SSL to be enabled? (default: true) " \
+    => 'menu_helper("agree", "foreman", "ssl")',
+    "Should Foreman run under apache & passenger? (default: true) " \
+    => 'menu_helper("agree", "foreman", "passenger")',
+    "Should Foreman be installed from the unstable repo? (default: false) " \
+    => 'menu_helper("agree", "foreman", "use_testing")',
```
Comment (domcleal) [inline generate_answers.rb]: This should change `repo` instead of `use_testing` now, ditto for proxy.

File: generate_answers.rb [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    => 'menu_helper("agree", "foreman_proxy", "tftp")',
+    "Should Foreman_proxy manage DNS? (default: false) " \
+    => 'menu_helper("agree", "foreman_proxy", "dns")',
+    "Should Foreman_proxy manage DHCP? (default: false) " \
+    => 'menu_helper("agree", "foreman_proxy", "dhcp")',
+  }
+end
+
+def puppet_questions
+  # this is only really here to provide a non-boolen example...
```
Comment (domcleal) [inline generate_answers.rb]: Useful example, but probably best to remove it.

File: generate_answers.rb (hunk 1 above)
Comment (GregSutcliffe) [inline generate_answers.rb]: Yep, will replace this as the non-boolean exmple ;)

**Comment by GregSutcliffe** [comment]
@domcleal done

**Pull request merged**